web/src/message_scroll.js

```diff
@@ -29,6 +42,16 @@ export function hide_scroll_to_bottom() {
             !$show_scroll_to_bottom_button.get(0).matches(":hover")
         ) {
             $show_scroll_to_bottom_button.removeClass("show");
+
+            // "Scroll Button" tooltip needs to be explicitly destroyed because the [data-reference-hidden] property does not work accurately under animation. Track the issue here - "https://github.com/atomiks/tippyjs/issues/1154".
+            $("#scroll-to-bottom-button-clickable-area").on("mouseenter", () => {
+                setTimeout(() => {
+                    const tippyInstance = $("#scroll-to-bottom-button-clickable-area")[0]._tippy;
+                    if (tippyInstance) {
+                        tippyInstance.destroy();
+                    }
+                }, 500);
+            });
```

Comment:
If we are doing the same thing twice, it should have a common method to do so.

Reply:
Sure ,updated.